CHANGELOG.md

```diff
@@ -1,23 +1,32 @@
 # Unreleased
- 
+
 ## New features
 
-### Replace back link placeholder URLs with javascript
+### Replace back link placeholder URLs with JavaScript
 
-Previously, users had to manually replace the `href` for each back link in the page templates.
+You can now use JavaScript to make the [back link component]( https://design-system.service.gov.uk/components/back-link/) take users to the previous page by default.
```

Comment:
@EoinShaughnessy I don't think this quite captures the change - we've made the back link in the kit use JavaScript *by default* so that it will 'just work' without users needing to do anything. Before, we had some placeholder text which users had to manually update to get the back link in their prototypes to work. This means they no longer need to do that.

Reply:
Ah I see - thanks. What about something like this?

```suggestion
We've added JavaScript to make the [back link component]( https://design-system.service.gov.uk/components/back-link/) take users to the previous page by default.
```

Reply:
@EoinShaughnessy That's great! 🎉